### PR TITLE
reason not mandatory when user is OOO for less than 3 days 

### PR DESCRIPTION
**Pull request by heyrandhir**
- [x] reason not mandatory when the user is OOO for less than 3 days 
- [x] added a new test case to handle this case

### CODE REVIEW

**Review by alishvagadiya — CHANGES_REQUESTED** [review]

File: test/fixtures/userStatus/userStatus.js [hunk 1]
```diff
@@ -12,6 +12,19 @@ const userStsDataForNewUser = {
   },
 };
 
+const userStsDataForOooStateForShortDuration = {
+  currentStatus: {
+    message: "",
```
[the remark this answers is not in the document]
Comment (heyrandhir) [reply test/fixtures/userStatus/userStatus.js:17]: If the user is OOO for less than 3 days then adding a message is not mandatory. I am testing the same. Hence left the it blank.

**Review by alishvagadiya — CHANGES_REQUESTED** [review]
Is there a test case when OOO duration is more than 3 days, and the reason is not mentioned, and failed/error Is the result, test cases are added?

If not please add test cases for that.

**Comment by heyrandhir** [comment]
> Is there a test case when OOO duration is more than 3 days, and the reason is not mentioned, and failed/error Is the result, test cases are added?
> 
> If not please add test cases for that.

Hi @alishvagadiya . Added the test case for it in the stacked PR which will be merged post this. This test can be found [here](https://github.com/Real-Dev-Squad/website-backend/pull/884/files#diff-734be2449d7d802ef008723d52ed7c5fae9f44665f12d97b6f89c6aad6166bc3R381-R397)

**Review by rohan09-raj — APPROVED** [review]
Nice job in handling the edge cases 💯

File: test/fixtures/userStatus/userStatus.js [hunk 2]
```diff
@@ -12,6 +12,19 @@ const userStsDataForNewUser = {
...
+    state: "OOO",
+    updatedAt: 1673893800000,
+    from: 1673893800000,
+    until: 1674066600000,
+  },
+  monthlyHours: {
+    updatedAt: 1668215609000,
+    committed: 40,
```
Comment (prakashchoudhary07) [inline test/fixtures/userStatus/userStatus.js:25]: Why are we passing monthlyHours when setting OOO?
Am I missing something here

**Review by prakashchoudhary07 — CHANGES_REQUESTED** [review]

File: test/fixtures/userStatus/userStatus.js (hunk 2 above)
Comment (heyrandhir) [reply test/fixtures/userStatus/userStatus.js:25]: this data I have used it here for testing if the endpoint is able to store the data in the collection document when requested by an authorized user or superuser. The test will work with both the partial status data as well as status data with monthly hours. Few of the tests are using complete data while few are using partial data.
